golden_response/staging/get_landing_page_data/asm.json

```diff
@@ -655,136 +651,8 @@
       "WorldBank_SP_DYN_IMRT_FE_IN": {},
       "WorldBank_SP_DYN_IMRT_IN": {},
       "WorldBank_SP_DYN_IMRT_MA_IN": {},
-      "WorldBank_SP_RUR_TOTL": {
```

Comment:
This is expected. After updating the chart config in g3, these should return under human readable names.